[2018-05-04 07:11:42] Acceso.INFO: Error en Inicio de Sesión: jluis Symfony\Component\Security\Core\Exception\BadCredentialsException: Bad credentials. in D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Core\Authentication\Provider\UserAuthenticationProvider.php:71 Stack trace: #0 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(145): session_start() #1 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(302): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->start() #2 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(249): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->getBag('attributes') #3 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(271): Symfony\Component\HttpFoundation\Session\Session->getBag('attributes') #4 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(73): Symfony\Component\HttpFoundation\Session\Session->getAttributeBag() #5 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall\ContextListener.php(93): Symfony\Component\HttpFoundation\Session\Session->get('_security_main') #6 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\WrappedListener.php(46): Symfony\Component\Security\Http\Firewall\ContextListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #7 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener.php(35): Symfony\Bundle\SecurityBundle\Debug\WrappedListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #8 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall.php(56): Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener->handleRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), Object(Symfony\Component\DependencyInjection\Argument\RewindableGenerator)) #9 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\EventListener\FirewallListener.php(48): Symfony\Component\Security\Http\Firewall->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #10 [internal function]: Symfony\Bundle\SecurityBundle\EventListener\FirewallListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #11 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\WrappedListener.php(104): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #12 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(212): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher)) #13 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(44): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #14 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #15 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(127): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #16 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #17 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #18 D:\wamp\www\gestion-foysi\web\app.php(19): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #19 {main} [] []
[2018-05-04 07:11:48] Acceso.INFO: Error en Inicio de Sesión: jluis Symfony\Component\Security\Core\Exception\BadCredentialsException: Bad credentials. in D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Core\Authentication\Provider\UserAuthenticationProvider.php:71 Stack trace: #0 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(145): session_start() #1 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(302): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->start() #2 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(249): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->getBag('attributes') #3 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(271): Symfony\Component\HttpFoundation\Session\Session->getBag('attributes') #4 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(73): Symfony\Component\HttpFoundation\Session\Session->getAttributeBag() #5 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall\ContextListener.php(93): Symfony\Component\HttpFoundation\Session\Session->get('_security_main') #6 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\WrappedListener.php(46): Symfony\Component\Security\Http\Firewall\ContextListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #7 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener.php(35): Symfony\Bundle\SecurityBundle\Debug\WrappedListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #8 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall.php(56): Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener->handleRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), Object(Symfony\Component\DependencyInjection\Argument\RewindableGenerator)) #9 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\EventListener\FirewallListener.php(48): Symfony\Component\Security\Http\Firewall->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #10 [internal function]: Symfony\Bundle\SecurityBundle\EventListener\FirewallListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #11 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\WrappedListener.php(104): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #12 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(212): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher)) #13 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(44): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #14 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #15 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(127): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #16 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #17 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #18 D:\wamp\www\gestion-foysi\web\app.php(19): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #19 {main} [] []
[2018-05-04 07:11:59] Acceso.INFO: Error en Inicio de Sesión: jluis Symfony\Component\Security\Core\Exception\BadCredentialsException: Bad credentials. in D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Core\Authentication\Provider\UserAuthenticationProvider.php:71 Stack trace: #0 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(145): session_start() #1 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(302): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->start() #2 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(249): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->getBag('attributes') #3 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(271): Symfony\Component\HttpFoundation\Session\Session->getBag('attributes') #4 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(73): Symfony\Component\HttpFoundation\Session\Session->getAttributeBag() #5 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall\ContextListener.php(93): Symfony\Component\HttpFoundation\Session\Session->get('_security_main') #6 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\WrappedListener.php(46): Symfony\Component\Security\Http\Firewall\ContextListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #7 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener.php(35): Symfony\Bundle\SecurityBundle\Debug\WrappedListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #8 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall.php(56): Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener->handleRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), Object(Symfony\Component\DependencyInjection\Argument\RewindableGenerator)) #9 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\EventListener\FirewallListener.php(48): Symfony\Component\Security\Http\Firewall->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #10 [internal function]: Symfony\Bundle\SecurityBundle\EventListener\FirewallListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #11 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\WrappedListener.php(104): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #12 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(212): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher)) #13 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(44): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #14 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #15 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(127): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #16 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #17 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #18 D:\wamp\www\gestion-foysi\web\app.php(19): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #19 {main} [] []
[2018-05-04 07:12:09] Acceso.INFO: Error en Inicio de Sesión: jluis Symfony\Component\Security\Core\Exception\BadCredentialsException: Bad credentials. in D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Core\Authentication\Provider\UserAuthenticationProvider.php:71 Stack trace: #0 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(145): session_start() #1 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(302): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->start() #2 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(249): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->getBag('attributes') #3 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(271): Symfony\Component\HttpFoundation\Session\Session->getBag('attributes') #4 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(73): Symfony\Component\HttpFoundation\Session\Session->getAttributeBag() #5 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall\ContextListener.php(93): Symfony\Component\HttpFoundation\Session\Session->get('_security_main') #6 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\WrappedListener.php(46): Symfony\Component\Security\Http\Firewall\ContextListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #7 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener.php(35): Symfony\Bundle\SecurityBundle\Debug\WrappedListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #8 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall.php(56): Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener->handleRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), Object(Symfony\Component\DependencyInjection\Argument\RewindableGenerator)) #9 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\EventListener\FirewallListener.php(48): Symfony\Component\Security\Http\Firewall->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #10 [internal function]: Symfony\Bundle\SecurityBundle\EventListener\FirewallListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #11 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\WrappedListener.php(104): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #12 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(212): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher)) #13 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(44): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #14 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #15 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(127): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #16 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #17 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #18 D:\wamp\www\gestion-foysi\web\app.php(19): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #19 {main} [] []
[2018-05-04 07:12:16] Acceso.INFO: Error en Inicio de Sesión: jluis Symfony\Component\Security\Core\Exception\BadCredentialsException: Bad credentials. in D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Core\Authentication\Provider\UserAuthenticationProvider.php:71 Stack trace: #0 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(145): session_start() #1 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php(302): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->start() #2 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(249): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->getBag('attributes') #3 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(271): Symfony\Component\HttpFoundation\Session\Session->getBag('attributes') #4 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Session.php(73): Symfony\Component\HttpFoundation\Session\Session->getAttributeBag() #5 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall\ContextListener.php(93): Symfony\Component\HttpFoundation\Session\Session->get('_security_main') #6 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\WrappedListener.php(46): Symfony\Component\Security\Http\Firewall\ContextListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #7 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener.php(35): Symfony\Bundle\SecurityBundle\Debug\WrappedListener->handle(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #8 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\Security\Http\Firewall.php(56): Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener->handleRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), Object(Symfony\Component\DependencyInjection\Argument\RewindableGenerator)) #9 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Bundle\SecurityBundle\EventListener\FirewallListener.php(48): Symfony\Component\Security\Http\Firewall->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #10 [internal function]: Symfony\Bundle\SecurityBundle\EventListener\FirewallListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #11 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\WrappedListener.php(104): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher)) #12 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(212): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher)) #13 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\EventDispatcher.php(44): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #14 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #15 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(127): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #16 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #17 D:\wamp\www\gestion-foysi\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #18 D:\wamp\www\gestion-foysi\web\app.php(19): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #19 {main} [] []
[2018-05-04 07:13:44] Acceso.INFO: Inicio de Sesión: Serrano Barrera, José Luis [] []
